ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-CareAgreement-01.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CarePlan xmlns="http://hl7.org/fhir">
+   <id value="nl-core-CareAgreement-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement"/>
+   </meta>
+   <status value="active"/>
+   <intent value="plan"/>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="288834001"/>
+         <display value="Agreeing on care plan"/>
+      </coding>
+   </category>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <period>
+      <start value="2020-09-15"/>
+   </period>
+   <contributor>
+      <reference value="PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role), J. van Dam, Psychologische zorgverleners, klinisch psycholoog, kinder- en jeugdpsycholoog NIP"/>
+   </contributor>
+   <addresses>
+      <reference value="Condition/nl-core-CareAgreement-01-Condition-01"/>
+      <type value="Condition"/>
+      <display value="Problem, type: Klinische bevinding"/>
+   </addresses>
+   <activity>
+      <detail>
+         <code>
+            <text value="Minimaal 3 keer per week dagboek in de E-Health app invullen."/>
+         </code>
+         <status value="in-progress"/>
+         <performer>
+            <reference value="Patient/nl-core-CareAgreement-01-Patient-01"/>
+            <type value="Patient"/>
+            <display value="Patient, A.S. van Dongen"/>
+         </performer>
+         <description value="POH-GGZ van de huisartspraktijk leest en reageert wekelijks (indien nodig)."/>
+      </detail>
+   </activity>
```

Comment:
The QA tools gives the following error:

  -  error at CarePlan.category[0] (9, 14):
     Slicing cannot be evaluated: Could not match discriminator ([coding]) for slice CarePlan.category:careAgreement in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement - the discriminator [coding] does not have fixed value, binding or existence assertions
  -  error at CarePlan.contributor[0] (24, 17):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.contributor[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-ContactPerson')): The URL 'PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.contributor[0] (24, 17):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.contributor[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole')): The URL 'PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.contributor[0] (24, 17):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.contributor[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient')): The URL 'PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (40, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-ContactPerson')): The URL 'Patient/nl-core-CareAgreement-01-Patient-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```
  -  error at CarePlan.activity[0].detail.performer[0] (40, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole')): The URL 'Patient/nl-core-CareAgreement-01-Patient-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (40, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient')): The URL 'Patient/nl-core-CareAgreement-01-Patient-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```